pkg/volume/util/util.go

```diff
@@ -582,3 +582,13 @@ func HasMountRefs(mountPath string, mountRefs []string) bool {
 	}
 	return false
 }
+
+//WriteVolumeCache executes a powershell command to flush disk data given the spcified mount path
+func WriteVolumeCache(deviceMountPath string, exec mount.Exec) error {
```

Comment:
this is in a OS agnostic util package. Could you make the function name/comment/implementation more Windows specific?

either `if runtime.GOOS == windows` do work otherwise just return nil.
or just name it something super Windows specific

Reply:
fixed